src/python/grpcio/grpc/_common.py

```diff
@@ -94,6 +99,19 @@ def decode(b):
       return b.decode('latin1')
 
 
+def channel_args(options):
+  if options is None:
+    return None
+  else:
+    channel_args = []
+    for key, value in options:
+      if isinstance(value, str_type):
```

Comment:
Why is `six.string_types` not the right value to use here instead of `str_type`?